Website/web.config

```diff
@@ -34,7 +34,7 @@
   </configSections>
   <connectionStrings>
     <!-- Connection String for SQL Server 2008/2012 Express -->
-    <add name="SiteSqlServer" connectionString="Data Source=.\MSSQL2017;Initial Catalog=HotcakesDev;User ID=uvuser;Password=uvpwd" providerName="System.Data.SqlClient"/>
+    <add name="SiteSqlServer" connectionString="Data Source=.\MSSQL2019;Initial Catalog=Hotcakes;User ID=dnn;Password=dnn" providerName="System.Data.SqlClient"/>
```

Comment:
Do not commit your web.config changes unless they need to be applied to all developers.  :)

Reply:
@VAlmea There hasn't been a response to this comment.  Please leave a comment telling me if this is resolved, or how else we might need to address this concern.